main.js

```diff
@@ -5,4 +5,82 @@ const navToggle = () => {
+
+    "The advertisement and application process for Opera Classica Europa's Opera Studio program, called the Opera Pathway.",
+
+    'A template intended at hosting music competitions, mostly those that were dedicated to a composer.  The Offenbach GP was such a competition later in time.',
+
+    'The advertisement and application process to enter the Callas Prize competition, which was supposed to take place at the Alte Oper Frankfurt for the finals.  Unfortunately cancelled due to COVID.',
+
+    "My personal singing portfolio and also the first site I ever coded.  In it's initial form it was merely a list, but later I created an interesting, albeit somewhat glitchy, navbar system.",
+
+  ],
+
+  liveButton: [
+    '<a href="https://www.offenbachgp.com/" rel=”noopener”>See Live <img src="images/liveIcon.png" alt="Live Icon"></a>',
+    '<a href="https://spoiledeggs.eu5.org/" rel=”noopener”>See Live <img src="images/liveIcon.png" alt="Live Icon"></a>',
+    '<a href="#" rel=”noopener”>See Live <img src="images/liveIcon.png" alt="Live Icon"></a>',
+    '<a href="#" rel=”noopener”>See Live <img src="images/liveIcon.png" alt="Live Icon"></a>',
+    '<a href="#" rel=”noopener”>See Live <img src="images/liveIcon.png" alt="Live Icon"></a>',
+    '<a href="#" rel=”noopener”>See Live <img src="images/liveIcon.png" alt="Live Icon"></a>',
+    '<a href="https://philipmodinos.ueuo.com" rel=”noopener”>See Live <img src="images/liveIcon.png" alt="Live Icon"></a>',
+
+  ],
+
+  sourceButton: [
+    '<a href="#" rel=”noopener”>See Source <img src="images/gitIcon.png" alt="Source Icon"></a>',
+    '<a href="#" rel=”noopener”>See Source <img src="images/gitIcon.png" alt="Source Icon"></a>',
+    '<a href="#" rel=”noopener”>See Source <img src="images/gitIcon.png" alt="Source Icon"></a>',
+    '<a href="#" rel=”noopener”>See Source <img src="images/gitIcon.png" alt="Source Icon"></a>',
+    '<a href="#" rel=”noopener”>See Source <img src="images/gitIcon.png" alt="Source Icon"></a>',
+    '<a href="#" rel=”noopener”>See Source <img src="images/gitIcon.png" alt="Source Icon"></a>',
+    '<a href="#" rel=”noopener”>See Source <img src="images/gitIcon.png" alt="Source Icon"></a>',
+  ],
+
+};
+
+function toggle(arrayNumber) {
+  const blur = document.getElementById('portfolio');
+  blur.classList.toggle('active');
+  const blur2 = document.getElementById('popUp');
+  blur2.classList.toggle('active');
+
+  document.getElementById('title').innerHTML = popUpInfo.title[arrayNumber];
+  document.getElementById('img').innerHTML = popUpInfo.image[arrayNumber];
+  document.getElementById('badges').innerHTML = popUpInfo.badges[arrayNumber];
+  document.getElementById('text').innerHTML = popUpInfo.text[arrayNumber];
+  document.getElementById('live').innerHTML = popUpInfo.liveButton[arrayNumber];
+  document.getElementById('source').innerHTML = popUpInfo.sourceButton[arrayNumber];
+}
+
+toggle();
```

Comment:
Dear Reviewer,
Thank you for your advice.  I will opt for the quick solution for now, as I am one day behind and I found the instruction quite lacking, which meant that to understand what I was supposed to do, I had to do a lot of extra research.  Not only was there a lack of instruction on how to implement objects, but there was also a lack of instruction on how to write the html and css (position: fixed in it's own container)... or at least I didn't see it, if it's there it's quite well hidden.  

I will however look to avoid this linter error from the start in future iterations of the project and other projects like it.